omniduct/duct.py

```diff
@@ -292,7 +292,7 @@ class name if not specified).
 
         atexit.register(self.disconnect)
         self.__prepared = False
-        self.__getting = False
```

Comment:
this is me nitting field naming. Not totally relevant for this PR.

Reply:
Hmmm... I think the original name makes more sense, but I'm probably biased (having written that code). This variable acts as an indicator for whether the `Duct` instance is currently preparing itself, and uses this information to prevent recursion during the preparation phase. The point is so that the code in `_prepare` can be written using normal self-attribute references, rather than using `object.__getattribute__` everywhere. The new name '__getting' seems a little misleading.

Reply:
hold up - previously the name was `__getting` which I found strange since it's indicating that the Duct is preparing ... hence the rename to `__preparing` ... which way are you for?